ada/ast.py

```diff
@@ -3298,6 +3363,16 @@ def env_names():
             Self.top_level_env_name.concat(String("__stub")).to_symbol
         ])
 
+    @langkit_property()
+    def previous_part_link_env_assoc():
+        return new_env_assoc(
+            key='__nextpart',
+            val=Self,
+            dest_env=Self.top_level_env_name.to_symbol.then(
+                lambda name: named_env(name)
+            )
```

Comment:
Couldn’t this be simplified to `named_env(Self.top_level_env_name.to_symbol)`?

Reply:
Yes! done